physionet-django/export/urls.py

```diff
@@ -6,6 +6,8 @@
 urlpatterns = [
     # API V1 Routes
     path('v1/project/published/', views.PublishedProjectList.as_view(), name='Published_project_list'),
+    path('v1/project/published/search/', views.PublishedProjectSearch.as_view(),
+         name='Published_project_search/'),
```

Comment:
These names should be lower case, I think (e.g. `published_project_search`). 

Can we fix the others at the same time?

Reply:
I have a bad habit of making the first characters upper case while typing. I've fixed the names to lower case and will keep it in mind from now on.